src/server/routes/map.js

```diff
@@ -0,0 +1,29 @@
+import express from 'express';
+
+const router = express.Router();
+
+router.get('/getmarkers', (req, res) => {
+  let markersData = [
```

Comment:
Please use "const" instead of "let" unless it needs to be reassigned: https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75